packages/editor-ui/src/components/Sticky.vue

```diff
@@ -246,14 +246,7 @@ export default defineComponent({
 			this.setForceActions(false);
 		},
 		async deleteNode() {
-			if (!this.data?.name) {
-				this.toast.showMessage({
-					title: this.$locale.baseText('nodeView.deleteSticky.error.title'),
-					message: this.$locale.baseText('nodeView.deleteSticky.error.message'),
-					type: 'error',
-				});
-				return;
-			}
+			assert(this.data?.name);
```

Comment:
I would have expected `assert(this.data)`

have you tested this? I don't believe this is a native browser function
I see Tomi used it in one place but he imported
`import { assert } from '@/utils/assert';`
https://github.com/n8n-io/n8n/blob/d361b42c7035a3edbdd999a322c9327a8f565f77/packages/editor-ui/src/components/CredentialPicker/CredentialPicker.vue#L5

should not this throw a type error if it's an unknown function?

Reply:
Ah, yeah. Not sure why it wasn't automatically imported 🤦‍♂️
Now, should throw an error like this if this ever happens (but will error out even when adding the Sticky if `data` is `null`, so as you said this is pretty impossible scenario)
![image](https://github.com/n8n-io/n8n/assets/2598782/4d7200a2-72a4-42a5-9444-99ad4e0e5b29)